unreleased/multi-package.md

```diff
@@ -0,0 +1,462 @@
+# Multi-Package CLI 
```

Comment:
I would think of this as a "command" rather than an entire "CLI," but I also think a more general title like "Multi-Package Builds" would be even better.